Dockerfile

```diff
@@ -1,24 +1,32 @@
-# Build Container
-FROM maven:3-amazoncorretto-15
+# Vue Build Container
+FROM node:lts-alpine as VUE
+WORKDIR /frontend
+COPY frontend .
+RUN npm install && npm run license-check && npm run license && npm run build
+
+# Micronaut build
+FROM maven:3-amazoncorretto-15 as MAVEN
 
 WORKDIR /home/maven/backend
 
 # Copy Files
 COPY backend/pom.xml backend/formatter.xml ./
 COPY backend/business-partner-agent ./business-partner-agent
 COPY backend/business-partner-agent-core ./business-partner-agent-core
-COPY frontend ../frontend
+# Copy Vue App
+COPY --from=VUE ["/frontend/licenses/[^attribution]*", "./business-partner-agent/src/main/resources/3rdPartyLicenses"]
+COPY --from=VUE /frontend/dist ./business-partner-agent/src/main/resources/public
 
```

Comment:
ok here there is more to be done, because the backend also does some attribution stuff in the frontend step and this is now disabled, we have to at least move this into a separate profile and activate it here again. Also we should generally discuss the build process and how we wat to structure it in the future.

Reply:
Yes, let's create a profile that does the necessary backend work and include it in this or a follow up PR. 

Is there backend work that needs to be done before frontend compiles?